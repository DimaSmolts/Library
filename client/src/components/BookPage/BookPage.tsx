import React from 'react';

interface BookNavParams {
  bookId: string;
}

export default class BookPage extends React.Component<BookNavParams> {
  render() {
    return <h1>Book #{this.props.bookId}</h1>
  }
}
