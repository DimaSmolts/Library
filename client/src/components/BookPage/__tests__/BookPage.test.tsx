import React from 'react'
import { shallow } from 'enzyme'

import BookPage from '../BookPage'

const bookId = '1'

test('snapshot: <BookPage /> renders correctly', () => {
  const tree = shallow(
    <BookPage bookId={bookId} />)

  expect(tree).toMatchSnapshot()
})
