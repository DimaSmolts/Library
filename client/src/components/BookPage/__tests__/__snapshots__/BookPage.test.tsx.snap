// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot: <BookPage /> renders correctly 1`] = `
<Fragment>
  <Connect(BookDetailsContainer)
    bookId="1"
  />
</Fragment>
`;
